src/gausskernel/storage/access/k2/pg_gate_api.cpp

```diff
@@ -304,6 +358,22 @@ K2PgStatus PgGate_InvalidateTableCacheByTableId(const char *table_uuid) {
   return K2PgStatus::NotSupported;
 }
 
+// Get ColumnSchema from column information
+k2pg::ColumnSchema getColumn(const std::string&& col_name, int order, K2SqlDataType k2pg_type, bool is_key, bool is_hash,  bool is_desc = false, bool is_nulls_first = false) {
```

Comment:
Should this be called "makeColumn" instead? Also for col_name I don't think a const lvalue ref makes sense, maybe just a const ref.